src/lib/core/styles/tokens/typography/_tokens.label.scss

```diff
@@ -26,6 +33,7 @@ $overline: utils.inherit-map(core.$base, (
 
 $tokens: (
   label: $label,
```

Comment:
We should rename "label" to "label1" if we're going to introduce a new "label2" style for consistency with our type scale. This will require a sweep of usages across the library.